src/service.js

```diff
@@ -230,22 +229,22 @@ async function startService({ peerId, currentPort, dispatcher, announceAddr } =
       announceAddr = peerAnnounceAddr
     }
 
-    const service = await libp2p.create({
+    const service = await createLibp2p({
       peerId,
       addresses: {
         listen: [`/ip4/0.0.0.0/tcp/${currentPort}/ws`],
         announce: announceAddr ? [announceAddr] : undefined
       },
-      modules: {
-        transport: [Websockets],
-        streamMuxer: [Multiplex],
-        connEncryption: [new Noise(null, null, noiseCrypto)]
-      }
+      transports: [new WebSockets()],
+      streamMuxers: [new Mplex()],
+      connectionEncryption: [new Noise()]
+      // TODO connectionEncryption: [new Noise(null, null, noiseCrypto)]
     })
 
-    service.on('error', err => {
-      logger.warn({ err }, `libp2p error: ${serializeError(err)}`)
-    })
+    // TODO?
+    // service.on('error', err => {
```

Comment:
Looks like `on` method is not available anymore

Reply:
you can do the following now https://github.com/libp2p/js-libp2p/blob/master/doc/migrations/v0.36-v.037.md#events